### ISSUE TITLE
`danger local` unknown option `--base`

### ISSUE THREAD

**Issue by adam-moss** [danger/danger-js#493]
Hi,

For `danger local --base develop --dangerfile dangerfile.js` I am getting:

```
error: unknown option `--base'

Error:  TypeError: Cannot read property 'concat' of undefined
    at Object.exports.mergeResults (/Users/adam.moss/projects/project_name/node_modules/danger/distribution/commands/utils/reporting.js:15:33)
    at Object.<anonymous> (/Users/adam.moss/projects/project_name/node_modules/danger/distribution/commands/utils/runDangerSubprocess.js:97:51)
    at step (/Users/adam.moss/projects/project_name/node_modules/danger/distribution/commands/utils/runDangerSubprocess.js:32:23)
    at Object.next (/Users/adam.moss/projects/project_name/node_modules/danger/distribution/commands/utils/runDangerSubprocess.js:13:53)
    at /Users/adam.moss/projects/project_name/node_modules/danger/distribution/commands/utils/runDangerSubprocess.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/adam.moss/projects/project_name/node_modules/danger/distribution/commands/utils/runDangerSubprocess.js:3:12)
    at ChildProcess.<anonymous> (/Users/adam.moss/projects/project_name/node_modules/danger/distribution/commands/utils/runDangerSubprocess.js:85:48)
    at ChildProcess.emit (events.js:160:13)
    at maybeClose (internal/child_process.js:943:16)
```

For branch structure:

master -> develop -> build/danger

**Comment by adam-moss** [comment]
Ok, so I think this is being caused by `--base` not being a valid option in [sharedDangerfileArgs.ts](source/commands/utils/sharedDangerfileArgs.ts)

Happy to submit a PR to include it if you want?

**Comment by orta** [comment]
Hrm, that's a not ideal - yeah, I'm surprised it should be available because I added it before the shared args:

https://github.com/danger/danger-js/blob/d89be2ad968a0098f5c855931858771f71125369/source/commands/danger-local.ts#L19

**Comment by adam-moss** [comment]
I think this is what is causing it: 
https://github.com/danger/danger-js/blob/d89be2ad968a0098f5c855931858771f71125369/source/commands/danger-runner.ts#L33

**Comment by orta** [comment]
Yes! 

The sub-process part, I see where you're coming from. Great thinking, I think there's a commander option to allow a command to ignore unknown input and this is a perfect case

**Comment by orta** [comment]
Maybe this: https://github.com/tj/commander.js/pull/138#issuecomment-29851220

**Comment by adam-moss** [comment]
Yep, could include that in danger-local and that should take care of it

**Comment by orta** [comment]
Want me to make the PR?

**Comment by adam-moss** [comment]
Happy either way 😄

**Comment by orta** [comment]
Lolz, I'll do it

**Comment by adam-moss** [comment]
Closed by #494
